backend/.env.example

```
@@ -5,3 +5,5 @@ SUPABASE_URL=""
 SUPABASE_SERVICE_ROLE_KEY=""
 # must be a securely generated random secret; keep it server-only and out of git
 ACTOR_KEY_SECRET="REPLACE_WITH_SECURE_RANDOM_SECRET"
+TRUST_RAILWAY_PROXY_HEADERS=false
+TRUSTED_PROXY_CIDRS=100.0.0.0/8
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRUSTED_PROXY_CIDRS key should go before the TRUST_RAILWAY_PROXY_HEADERS key

(UnorderedKey)